src/System Application/App/AI/src/Azure OpenAI/AzureOpenAIImpl.Codeunit.al

```diff
@@ -47,6 +47,7 @@ codeunit 7772 "Azure OpenAI Impl" implements "AI Service Name"
         TelemetryMetapromptRetrievalErr: Label 'Unable to retrieve metaprompt from Azure Key Vault.', Locked = true;
         TelemetryFunctionCallingFailedErr: Label 'Function calling failed for function: %1', Comment = '%1 is the name of the function', Locked = true;
         AzureOpenAiTxt: Label 'Azure OpenAI', Locked = true;
+        BillingTypeAuthorizationErr: Label 'Usage of AI resources not authorized with chosen billing type. Please reach out to your administrator to resolve this issue.';
```

Comment:
Add the billing type and the capability to the error. That will make it easier for the enduser/admin to understand the error. It'll be more easily actionable.